src/pages/overview/pdf-accessibility-auto-tag-api/quickstarts/dotnet/index.md

```diff
@@ -0,0 +1,333 @@
+---
+title: .NET | Quickstarts | PDF Accessibility Auto Tag API | Adobe PDF Services
+---
+
+# Quickstart for PDF Accessibility Auto Tag API (.NET)
+
+To get started using Adobe PDF Accessibility Auto Tag API, let's walk through a simple scenario - taking an input PDF document and running PDF Accessibility Auto Tag API against it. Once the PDF has been tagged, we'll provide the document with tags and optionally, a report file. In this guide, we will walk you through the complete process for creating a program that will accomplish this task. 
+
+## Prerequisites
+
+To complete this guide, you will need:
+
+* [.NET Core: version 2.1 or above](https://dotnet.microsoft.com/en-us/download)
+* [.Net SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+* A build tool: Either Visual Studio or .NET Core CLI.
+* An Adobe ID. If you do not have one, the credential setup will walk you through creating one.
+* A way to edit code. No specific editor is required for this guide.
+  
+## Step One: Getting credentials
+
+1) To begin, open your browser to <https://documentservices.adobe.com/dc-integration-creation-app-cdn/main.html?api=pdf-extract-api>. If you are not already logged in to Adobe.com, you will need to sign in or create a new user. Using a personal email account is recommend and not a federated ID.
+
+![Sign in](./shot1.png)
+
+2) After registering or logging in, you will then be asked to name your new credentials. Use the name, "New Project". 
+
+3) Change the "Choose language" setting to ".Net". 
+
+4) Also note the checkbox by, "Create personalized code sample." This will include a large set of samples along with your credentials. These can be helpful for learning more later. 
+
+5) Click the checkbox saying you agree to the developer terms and then click "Create credentials."
+
+![Project setup](./shot2.png)
+
+6) After your credentials are created, they are automatically  downloaded:
+
+![alt](./shot3.png)
+
+## Step Two: Setting up the project
+
+1) In your Downloads folder, find the ZIP file with your credentials: PDFServicesSDK-.NetSamples.zip. If you unzip that archive, you will find a README file, your private key, and a folder of samples:
+
+![alt](./shot5.png)
+
+2) We need two things from this download. The `private.key` file (as shown in the screenshot above, and the `pdfservices-api-credentials.json` file. You can find this in the `adobe-DC.PDFServicesSDK.NET.Samples` folder, inside any of the sample subdirectories, so for example, the `CombinePDF` folder.
+
+![alt](./shot6.png)
+
+<InlineAlert slots="text" />
+
+Note that that private key is *also* found in this directory so feel free to copy them both from here.
+
+3) Take these two files and place them in a new directory.
+
+4) In your new directory, create a new file, `ExtractTextInfoFromPDF.csproj`. This file will declare our requirements as well as help define the application we're creating.

```

Comment:
`ExtractTextInfoFromPDF` file is taken in ref to Extract PDF samples, which should be updated for Autotag use-case.